indico/modules/rb_new/client/js/common/rooms/EquipmentList.jsx

```diff
@@ -0,0 +1,95 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2018 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import _ from 'lodash';
+import {Dropdown, Icon, List} from 'semantic-ui-react';
+import * as roomsSelectors from './selectors';
+
+import './EquipmentList.module.scss';
+
+
+class EquipmentList extends React.Component {
+    static propTypes = {
+        onChange: PropTypes.func.isRequired,
+        onFocus: PropTypes.func.isRequired,
+        onBlur: PropTypes.func.isRequired,
+        value: PropTypes.array.isRequired,
+        label: PropTypes.string.isRequired,
+        equipmentTypes: PropTypes.arrayOf(PropTypes.object).isRequired,
+    };
+
+    generateEquipmentOptions = () => {
```

Comment:
I think you could make this a getter since it doesn't have args, that way you could just call it as `this.equipmentOptions`